optaweb-vehicle-routing-frontend/package-lock.json

```diff
@@ -5,16 +5,12 @@
   "dependencies": {
     "@babel/code-frame": {
       "version": "7.0.0",
-      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
```

Comment:
Do not modify `package-lock.json`.

Reply:
that was done by vscode ... gonna revert that change

Reply:
@yurloc done

Reply:
Perfect! Thanks :slightly_smiling_face: